metadata-ingestion/tests/integration/kafka-connect/test_kafka_connect.py

```diff
@@ -252,3 +253,76 @@ def test_kafka_connect_ingest(docker_compose_runner, pytestconfig, tmp_path, moc
             golden_path=test_resources_dir / "kafka_connect_mces_golden.json",
             ignore_paths=[],
         )
+
+
+@freeze_time(FROZEN_TIME)
+@pytest.mark.integration_batch_1
+def test_kafka_connect_mongosourceconnect_ingest(
```

Comment:
@shirshanka I separated the mongo source connector test from the original. The test passed locally, but for some reason it fails in CI. It's bit hard to debug, but either the collection in Mongo does not get created as it should, or there is some race condition that causes the connector to not be ready when the ingestion is run. 

What do you think we should do? Skip the entire test? Comment out the aspects related to the topics and only test the existence of the connector?